openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -324,6 +324,29 @@ def get_from_ids_if_exists(cls, course_ids):
             )
         }
 
+    @classmethod
+    def get_from_id_if_exists(cls, course_id):
+        """
+        Return a CourseOverview for the provided course_id if it exists.
+        Returns None if no CourseOverview exists with the provided course_id
```

Comment:
Why choose to return `None` instead of just raising the exception?

Reply:
From a dev perspective I'd expect `get_from_id` to return an exception if it doesn't exist but `get_from_id_if_exists` to me indicates that the method will catch the exception and just return None.

Reply:
ok.